entity-framework/core/modeling/bulk-configuration.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Model Bulk Configuration - EF Core
+description: How to apply bulk configuration during model building in Entity Framework Core
+author: AndriySvyryd
+ms.date: 11/05/2021
+uid: core/modeling/bulk-configuration
+---
+# Model bulk configuration
+
+When an aspect needs to be configured in the same way across multiple entity types the following techniques allow to reduce code duplication and consolidate the logic.
+
+See the [full sample project](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Modeling/BulkConfiguration) containing the code snippets presented below.
+
+## Metadata API
```

Comment:
I understand that this is about using the (regular) metadata API for bulk config, but the actual focus of this section seems to be "Bulk configuration in OnModelCreating" (the metadata API is just a means?)

```suggestion
## Bulk configuration in OnModelCreating
```